packages/ButtonGroup/src/button-group.scss

```diff
@@ -83,6 +88,10 @@
     margin-left: -1px;
   }
 
+  > svg {
+    margin-right: var(--ids-button-group-icon-spacing);
```

Comment:
The margin right is always there, even when there is no text (except for compact ones). There should be no margin right if there is no text. It could maybe be written a different way. For example, put margin left on the text instead, but only when the svg is there. svg + .text {margin-right:}